x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/index.tsx

```diff
@@ -53,28 +53,31 @@ const missingFieldLabel = i18n.translate('xpack.lens.indexPattern.missingFieldLa
   defaultMessage: 'Missing field',
 });
 
-function ofName(name?: string, count: number = 0, rare: boolean = false) {
+function ofName(name?: string, secondaryFieldsCount: number = 0, rare: boolean = false, termsSize: number = 0) {
   if (rare) {
     return i18n.translate('xpack.lens.indexPattern.rareTermsOf', {
-      defaultMessage: 'Rare values of {name}',
+      defaultMessage: 'Rare {termsSize} values of {name}',
```

Comment:
rare terms don't use the size, we shouldn't include it in the default label for those